aspnetcore/security/enforcing-ssl.md

```diff
@@ -184,19 +217,17 @@ The preceding example shows how to add additional hosts.
 
```

Comment:
" `UseHsts` excludes the following loopback hosts by default:"

~~The preceding example shows how to add additional hosts.~~ Redundant.